task_geo/common/fips_codes.py

```diff
@@ -0,0 +1,3296 @@
+    '55137': 'Waushara',
+    '55139': 'Winnebago',
+    '55141': 'Wood',
+    '56001': 'Albany',
+    '56003': 'Big Horn',
+    '56005': 'Campbell',
+    '56007': 'Carbon',
+    '56009': 'Converse',
+    '56011': 'Crook',
+    '56013': 'Fremont',
+    '56015': 'Goshen',
+    '56017': 'Hot Springs',
+    '56019': 'Johnson',
+    '56021': 'Laramie',
+    '56023': 'Lincoln',
+    '56025': 'Natrona',
+    '56027': 'Niobrara',
+    '56029': 'Park',
+    '56031': 'Platte',
+    '56033': 'Sheridan',
+    '56035': 'Sublette',
+    '56037': 'Sweetwater',
+    '56039': 'Teton',
+    '56041': 'Uinta',
+    '56043': 'Washakie',
+    '56045': 'Weston',
+    '60010': 'Unknown',
+    '60020': 'Manua',
+    '60050': 'Tutuila',
+    '66010': 'Cocos Island',
+    '69100': 'Rota',
+    '69110': 'Saipan',
+    '69120': 'Aguijan',
+    '72001': 'Adjuntas',
+    '72003': 'Aguada',
+    '72005': 'Aguadilla',
+    '72007': 'Aguas Buenas',
+    '72009': 'Aibonito',
+    '72011': 'Anasco',
+    '72013': 'Arecibo',
+    '72015': 'Arroyo',
+    '72017': 'Barceloneta',
+    '72019': 'Barranquitas',
+    '72021': 'Bayamon',
+    '72023': 'Cabo Rojo',
+    '72025': 'Caguas',
+    '72027': 'Camuy',
+    '72029': 'Canovanas',
+    '72031': 'Carolina',
+    '72033': 'Catano',
+    '72035': 'Cayey',
+    '72037': 'Ceiba',
+    '72041': 'Cidra',
+    '72043': 'Coamo',
+    '72045': 'Comerio',
+    '72047': 'Corozal',
+    '72049': 'Culebra',
+    '72051': 'Dorado',
+    '72053': 'Fajardo',
+    '72054': 'Florida',
+    '72055': 'Guanica',
+    '72057': 'Guayama',
+    '72059': 'Guayanilla',
+    '72061': 'Guaynabo',
+    '72063': 'Gurabo',
+    '72065': 'Hatillo',
+    '72067': 'Hormigueros',
+    '72071': 'Isabela',
+    '72073': 'Jayuya',
+    '72075': 'Juana Diaz',
+    '72077': 'Juncos',
+    '72079': 'Lajas',
+    '72081': 'Lares',
+    '72083': 'Las Marias',
+    '72085': 'Las Piedras',
+    '72087': 'Loiza',
+    '72089': 'Luquillo',
+    '72091': 'Manati',
+    '72093': 'Maricao',
+    '72095': 'Maunabo',
+    '72097': 'Mayaguez',
+    '72099': 'Moca',
+    '72101': 'Morovis',
+    '72103': 'Nabuabo',
+    '72105': 'Naranjito',
+    '72107': 'Orocovis',
+    '72109': 'Patillas',
+    '72111': 'Penuelas',
+    '72113': 'Ponce',
+    '72115': 'Quebradillas',
+    '72117': 'Rincon',
+    '72119': 'Rio Grande',
+    '72121': 'Sabana Grande',
+    '72123': 'Salinas',
+    '72125': 'San German',
+    '72127': 'San Juan',
+    '72129': 'San Lorenzo',
+    '72131': 'San Sabastian',
+    '72133': 'Santa Isabel',
+    '72135': 'Toa Alta',
+    '72137': 'Toa Baja',
+    '72139': 'Trujillo Alto',
+    '72141': 'Utuado',
+    '72143': 'Vega Alta',
+    '72145': 'Vega Baja',
+    '72147': 'Vieques',
+    '72149': 'Villalba',
+    '72151': 'Yabucoa',
+    '72153': 'Yauco',
+    '78010': 'St. Croix',
+    '78020': 'St. John',
+    '78030': 'St. Thomas',
+}
+
+
+def county_fips_to_name(fips_code):
+    return COUNTY_FIPS_CODES[fips_code] if fips_code in COUNTY_FIPS_CODES else 'Unknown'
```

Comment:
This is bad practice:

1. If the value is not present, you should return `np.nan` or `None` instead of `Unknown`
2. The behavior of what you are doing with this if/else can be replaced with [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get), like this:

```python
return COUNTRY_FIPS_CODES.get(fips_code)
```